api/HTMLIFrameElement.json

```diff
@@ -211,24 +211,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/allowPaymentRequest",
           "support": {
             "chrome": {
-              "version_added": "60",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "#web-payments",
-                  "value_to_set": "Enabled"
-                }
-              ]
+              "version_added": "60"
             },
             "chrome_android": {
-              "version_added": "61",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "#web-payments",
-                  "value_to_set": "Enabled"
-                }
-              ]
+              "version_added": "57"
```

Comment:
The difference to desktop seemed a bit suspect at first, but looking at `PaymentRequest` it seems like the feature as a whole reached Android before desktop.